Dockerfile

```diff
@@ -5,8 +5,11 @@ ENV PYTHONUNBUFFERED 1
 
 WORKDIR /moto/
 RUN  pip3 --no-cache-dir install --upgrade pip setuptools && \
-     pip3 --no-cache-dir install ".[server]"
+     pip3 --no-cache-dir install ".[server]" && \
+     python3 setup.py develop && \
```

Comment:
The changes here are a bit confusing to me - do we need them?
`pip install .` already install all the runtime dependencies, and we shouldn't need the dev-dependencies at this point.
The make-command is not used inside Docker/while running `moto/server.py`, so that's also unnecessary.

On a general note: This Docker-file is mainly used by the maintainers to create our releases, so I'd like to keep this intact if possible. If it does need changing to use this as a development server, maybe we can have a separate Dockerfile for that purpose? (I'm not too familiar with Docker, so if there is a better way to ensure we can serve both use-cases, I'm all ears.)